spec/omniauth/strategies/redbooth_spec.rb

```diff
@@ -1,102 +1,118 @@
 require 'spec_helper'
 
-describe OmniAuth::Strategies::Redbooth do
+RSpec.describe OmniAuth::Strategies::Redbooth do
```

Comment:
In the past `RSpec` added this methods `describe` to the global namespace, `Kernel` (or maybe `Object`?), but that is not really cool. In the latest versions of `RSpec` you can remove this feature and use `RSpec` to open the spec definitions, as you can see in this example.

I would say is better to keep it, is more explicit.